src/Tribe/API.php

```diff
@@ -454,11 +454,25 @@ protected static function prepare_event_date_meta( $event_id, $data ) {
 
 			// Additionally store datetimes in UTC
 			if ( empty( $data['EventStartDateUTC'] ) ) {
-				$data['EventStartDateUTC'] = Tribe__Events__Timezones::to_utc( $data['EventStartDate'], $data['EventTimezone'] );
+				if ( Tribe__Events__Timezones::is_utc_offset( $data['EventTimezone'] ) ) {
+					// Convert a UTC offset timezone to a localized timezone, e.g. UTC-5 to America/New_York.
+					$data['EventStartDateUTC'] = Tribe__Date_Utils::immutable( $data['EventStartDate'], $data['EventTimezone'] )
+					                                              ->setTimezone( new DateTimeZone( 'UTC' ) )
```

Comment:
Collect the result of `new DateTimeZone( 'UTC' )` in a variable to avoid building it twice.